app/osparc.py

```diff
@@ -1,71 +1,159 @@
-from app.config import Config
 import json
 import osparc
 import tempfile
+
+from app.config import Config
+from flask import abort
+from osparc.rest import ApiException
 from time import sleep
 
 
+OPENCOR_SOLVER = "simcore/services/comp/opencor"
+DATASET_4_SOLVER = "simcore/services/comp/rabbit-ss-0d-cardiac-model"
+DATASET_17_SOLVER = "simcore/services/comp/human-gb-0d-cardiac-model"
```

Comment:
I can't see how it would. Here, we really want to distinguish between CellML-based simulation datasets (which require `OPENCOR_SOLVER` to be solved) and SIM-Core based simulation datasets (which require another solver). In the latter case, to have `DATASET_4_SOLVER`, `DATASET_17_SOLVER`, and `DATASET_78_SOLVER` is just so that we can provide a more accurate user feedback, should we ever try to run a simulation using a solver that is not supported.